client/dist/style.css

```diff
@@ -86,23 +74,131 @@
   padding-top: 25px;
 }
 
-.colored {
-  color: #da3743;
-}
+/*CSS for stars*/
 
 .stars-component {
   display: flex;
   flex-direction: row;
-  padding-bottom: 10px;
+  padding-bottom: 4%;
+}
+
+.colored {
+  color: #da3743;
 }
 
 .uncolored {
   color: #6f737b;
 }
 
-.icon-container {
+/*CSS for header*/
+
+.header-container {
+  font-family: 'Josefin Sans', sans-serif;
+  font-weight: bold;
+  margin-left: 10%;
+  margin-right: 10%;
+  margin-top: 15px;
+  border-bottom: 1px solid #2d333f;
+  border-top: 1px solid #2d333f;
+  padding-left: 5%;
+  padding-top: 10px;
+  padding-bottom: 10px;
```

Comment:
Padding shorthand, same as margin shorthand, can be simplified as:
`padding: 10px 0 10px 5%;`